### PR TITLE
Remove HTTP part of the client for API v6 services

### PR DESCRIPTION
**Pull request by palczyn**
Issue: KNTK 370
Firstly this PR clears repo of depracated HTTP services for API v6.
In addition some improvements can be found for timeout tests and gRPC examples in separate commits.

### CODE REVIEW

**Comment by mateuszmidor** [comment]
While running the users example, I get back an error:
`go test -tags examples -count 1 -v -run Users github.com/kentik/community_sdk_golang/examples`
```
go: finding module for package github.com/kentik/community_sdk_golang/kentikapi/synthetics
# github.com/kentik/community_sdk_golang/examples
examples/synthetics_example_test.go:14:2: no matching versions for query "latest"
FAIL    github.com/kentik/community_sdk_golang/examples [setup failed]
FAIL
```
It works fine on master.

File: examples/synthetics_example_test.go [hunk 1]
```diff
@@ -546,11 +554,11 @@ func runGRPCListTests(ctx context.Context, client *kentikapi.Client) error {
 	return nil
 }
 
-func runGRPCGetHealthForTests(ctx context.Context, client *kentikapi.Client, testIDs []string) error {
+func runGRPCGetHealthForTests(ctx context.Context, client *kentikapi.Client, testID string) error {
```
Comment (mateuszmidor) [inline examples/synthetics_example_test.go]: No longer GetHealthForTests, just single Test now

File: examples/synthetics_mesh_example_test.go [hunk 2]
```diff
@@ -284,3 +285,26 @@ func (m metricsMatrixGRPC) getMetricGRPC(fromAgent string, toAgent string) (*syn
...
+	}
+
+	if getAllResp.Tests != nil {
+		for _, test := range getAllResp.GetTests() {
+			if test.GetType() == "application_mesh" {
+				return test.GetId(), nil
+			}
+		}
+	}
+	return "", fmt.Errorf("No tests with type application_mesh: %v", err)
```
Comment (mateuszmidor) [inline examples/synthetics_mesh_example_test.go]: I would include information that there is no such tests but specifically for requested Kentik Account 
err here is most likely always nil?

**Review by mateuszmidor — CHANGES_REQUESTED** [review]
There is an issue with running examples the way README suggests

**Review by mateuszmidor — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]

File: examples/synthetics_mesh_example_test.go [hunk 3]
```diff
@@ -14,157 +13,21 @@ import (
...
 
 func runGetMeshTestResultsGRPC() error {
-	flag.Parse()
+	testID, err := pickTestID()
+	if err != nil {
+		return err
+	}
 
-	mesh, err := getMeshTestResultsGRPC(*testID)
+	mesh, err := getMeshTestResultsGRPC(testID)
```
Comment (danielfurman) [inline examples/synthetics_mesh_example_test.go:30]: -> `getMeshTestResults`
There is no need for gRPC suffix. In other places in examples/tests too.

**Pull request merged**